src/azure-cli/HISTORY.rst

```diff
@@ -11,6 +11,9 @@ Release History
 
 * Update ADLS version (0.0.45).
 
+**Profile**
+* Suppress `--subscription` argument for logout command.
```

Comment:
Should this be annotated as a breaking change?  Technically any scripts that might be using it would break, but then it didn't work as expected to begin with.

Reply:
Yeah, I'm not too concerned about this. The only people it would "break" haven't been working anyway. I don't think this needs any escalated attention. Thoughts, @yugangw-msft?